api/src/apps/hooks.rs

```diff
@@ -252,7 +260,7 @@ mod tests {
         let script = r#"
         function deploymentHook( appName, configs ) {
             if( appName === 'master' ) {
-                return configs.filter( service => service.name !== 'service-b' );
+                return configs.filter( service => service.name !== 'service-c' );
```

Comment:
Why did you change this test? I would that the tests can be keep as they were before upgrading Boa

Reply:
This has been reverted. Commits are squashed.